wiki/Tournaments/oPHN/2023_mania/en.md

```diff
@@ -0,0 +1,640 @@
+
+## Ruleset
+
+### Tournament organization
+
+#### Gamemode and introduction
+
+As the tournament name implies, this is an osu!mania tournament set up as a national championship for the Philippine osu!mania community. This tournament accommodates both 4K and 7K as separate tournaments.
+
+#### General eligibility rules
+
+Players who currently have (or had at some point since account creation) the Philippine flag on their profile or players who do not have the Philippine flag on their profiles but **can present a proof of their Philippine citizenship** to the staff are the only ones allowed to join the tournament.
+
+Staff members who are **[not referees, mappool selectors, hosts, nor are part of any managing role](/wiki/Tournaments/Official_support#staff)** are permitted to join the tournament.
+
+#### Round overview
+
+With the exception of the qualification stage (which does not have a winning condition), each tournament round will have a selectable mappool of a certain size, a tiebreaker song, and a winning condition. The format is as follows:
+
+| Stage | Format | Mappool size |
+| :-- | :-- | :-- |
+| Round of 32 | Best of 9 | 12 FM, 1 TB |
+| Round of 16 | Best of 9 | 12 FM, 1 TB |
+| Quarterfinals | Best of 11 | 14 FM, 1 TB |
+| Semifinals | Best of 11 | 14 FM, 1 TB |
+| Finals | Best of 13 | 16 FM, 1 TB |
+| Grand Finals | Best of 13 | 16 FM, 1 TB |
+
+Mods that are allowed to be used in matches, if the player opts to use them, are Hidden, Flashlight, Fade In, Mirror, and No Fail. In addition, all maps will be played with ScoreV2.
+
+#### Qualification process
+
+All players that enter the tournament will be eligible to play in the qualification round. However, the number of participants who will advance to the next round will be cut down to the nearest power of 2. The maximum number of players who can advance to the bracket stage is 32. The criterion used to determine whether a player will progress to the next round is their performance in the qualification stage.
+
+The qualification round will have a 6-song mappool that widely varies in skillsets tested. These tested skillsets will correspond to skillsets commonly seen in mappools throughout the tournament.
+
+**All six notecharts will be created by the mappool selectors and/or special guest notecharters and will be unreleased prior to the tournament qualification stage, to provide zero opportunity for advantage through familiarity.**
+
+All players will have two chances at each qualifier notechart, in which the highest of the two runs on each qualifier notechart will be used to calculate your final score.
+
+Players will be organized into groups based on their availability. Because qualification is not a process that involves 1 on 1 matchups, it is actually possible to have 8 players go through the qualification process at once. If this is a confusing process, do not worry: simply adhere to the schedule that will be released and follow referee instructions during the match.
```

Comment:
```suggestion
Players will be organised into groups based on their availability. Because qualification is not a process that involves 1 on 1 matchups, it is actually possible to have 8 players go through the qualification process at once. If this is a confusing process, do not worry: simply adhere to the schedule that will be released and follow referee instructions during the match.
```